onnxruntime/core/providers/acl/activation/activations.cc

```diff
@@ -0,0 +1,57 @@
+// Copyright(C) 2018 Intel Corporation
+// Copyright (c) 2019, NXP Semiconductor, Inc. All rights reserved.
+// Licensed under the MIT License
+
+#ifdef _WIN32
+#pragma warning(disable : 4244)
+#endif
+
+#include "core/providers/acl/acl_common.h"
+#include "core/providers/acl/activation/activations.h"
+#include "core/providers/acl/acl_fwd.h"
+
+// ACL
+#include "arm_compute/runtime/Tensor.h"
+#include "arm_compute/runtime/TensorAllocator.h"
+
+// NEON
+#include "arm_compute/runtime/NEON/functions/NEActivationLayer.h"
+
+namespace onnxruntime {
+namespace acl {
+
+template <typename T>
+Status Relu<T>::Compute(OpKernelContext* context) const {
+
+	arm_compute::Tensor in, out;
+	arm_compute::NEActivationLayer layer;
+
+        const Tensor* X = context->Input<Tensor>(0);
```

Comment:
nit: formatting seems off here? 
note, we have a .clang-format file in root of repo.

Reply:
Thanks!